src/services/YotiSessionCompletionProcessor.ts

```diff
@@ -212,18 +212,29 @@ export class YotiSessionCompletionProcessor {
   				throw new AppError(HttpCodesEnum.SERVER_ERROR, "Missing Name Info in DocumentFields");
   			}
 
-  			if (missingGivenName || missingFamilyName) {
```

Comment:
wait sorry remind me why we have moved this if?

Reply:
To catch it early and log if the names are missing even if they're present in DB and we can use those to create VC